README.md

```diff
@@ -1 +1,102 @@
-# deployment-deploy-services
\ No newline at end of file
+# How to deploy dalite-ng on OpenStack server
+ 
+This repository contains ansible deployment scripts allowing one to deploy dalite-ng to OpenStack server.
+
+
+Create a OpenStack environment
+------------------------------
+
+1. Create a root volume that has 40GB size. It should be based on Ubuntu 14.04 image. 
+   We suggest you use vanilla image from [ubuntu page](https://cloud-images.ubuntu.com/). 
+2. Create `vps-ssd-3` instance from this volume: (`Boot Source` -> `Volume` -> `Your root volume`). 
```

Comment:
I'm not sure this does what you think it does - If you create a volume based on the stock ubuntu image, and then boot from that volume, it will just run the installation from the volume, not install the system on it?